book/run/pruning.md

```diff
@@ -72,6 +68,42 @@ If pruned fully, this is the total freed space you'll get, per part:
 | Account History    | 230GB |
 | Storage History    | 680GB |
 
+### Full Node
+
+Full node occupies at least 950GB.
```

Comment:
comment about the peak database needed for sync would be helpful here

Reply:
950GB is peak